README.md

```diff
@@ -38,6 +38,70 @@ Discuss the ideas with the community.
 * Reddit - https://www.reddit.com/r/haskell/comments/sogi3s/learn_you_a_haskell_a_community_version/
 * Haskell Discourse - https://discourse.haskell.org/t/learn-you-a-haskell-a-community-version/4056
 
+## Contributing
+
+We are happy to get your contributions!
+For the most part, you can simply edit Markdown files and open a PR with the edits.
+If you want to preview the changes locally, you need to build the site as discussed below.
+
+### Building the site
+
+The site is built using [Hakyll](https://jaspervdj.be/hakyll/), a static site generator in the form of a Haskell library.
+With Hakyll, you first build a Haskell application `site`, and then run it to generate HTML for the website.
```

Comment:
This should make a note that the two steps can be combined in some workflows (e. g. `cabal run`).